test-resources/openapi-service-specs/test-service.json

```diff
@@ -661,6 +679,9 @@
       "OtherSimpleTestEntity": {
         "$ref": "#/components/schemas/SimpleTestEntity"
       },
+      "simple-test-entity«WITH_Symbols»": {
```

Comment:
May be add a test with a schema "simple<test<entity<WITH_Symbols<" so the generated type will be the same as this instance, although it seems to be tested in unit test.

Reply:
Done.